src/DatePicker/DatePicker.js

```diff
@@ -309,6 +311,28 @@ class DatePicker extends Component {
         this.validateDates(this.props.onBlur);
     };
 
+    _showTodayHeader = () => {
```

Comment:
let's remove the `_` these would be the only class methods that have them in the repo

Reply:
Changed.